@shared/api/helpers/popupStoreFacade.ts

```diff
@@ -0,0 +1,9 @@
+/* 
+  This object can store data from `popup` without having to pass it as function params.
+  Generally, we prefer passing data as function params, but there are cases where we want to transmit
+  lots of data on every call. This is is a way to do that.
+*/
+
+export const popupStoreFacade = {
```

Comment:
By hanging the popup's redux store off of this object, the background always has access to the current state of the store. It can just call `popupStoreFacade.currentStore.getState()` to get the full state of the popup's redux store. 

I went over a few different options on how to do this:

One option was to send the store/parts of the store along with every request to the background. I moved away from this for the reasons mentioned above about having to manually include this on every request.

I also tried refactoring to move the entire redux store to the `@shared` folder, but that ended up becoming a huge refactor.

I settled on this is as the least invasive. @CassioMG @aristidesstaffieri Lmk what you think!

Reply:
Talked with @aristidesstaffieri and we came up with a better way to do this in: https://github.com/stellar/freighter/pull/1848/commits/47b4898c5d140b06046e5e51c53e7c9a31cb1093